cluster-agent/Makefile

```diff
@@ -140,3 +141,18 @@ package_test:
 	sudo apt-get purge -y cluster-agent
 
 tarball: common-tarball
+
+deb-push:
+	if [ -z "$(cat VERSION | grep 'dev')" ]; then \
+		echo "Uploading artifacts..."; \
+		cd build/artifacts; \
+		for DEB_PKG in *.deb; do \
+			PKG_VER=$(dpkg-deb -f "$${DEB_PKG}" Version); \
+			PKG_NAME=$(dpkg-deb -f "$${DEB_PKG}" Package); \
+			REPOSITORY=en/deb/$${PKG_NAME}; \
+			aws ecr create-repository --region us-west-2 --repository-name edge-orch/$${REPOSITORY} || true; \
```

Comment:
This create command has been removed in other Makefiles, it should not be included here but as part of the workflow itself if needed.